### PR TITLE
Merge release/19.7 after code freeze

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] New empty entry created in `RELEASE-NOTES.txt` for next iteration.
- [x] `{jetpack_}metadata/release_notes.txt` extracted for WordPress and Jetpack.
- [x] `WordPress/jetpack_metadata/release_notes.txt` audited to remove any item not applicable for Jetpack.
- [x] Internal dependencies (FluxC, Login. Stories. About…) bumped to a new stable version in `build.gradle` if necessary.
- [x] `WordPress/src/main/res/values/strings.xml` updated with strings from binary dependencies.
- [x] New strings frozen/copied into `fastlane/resources/values/strings.xml`.
- [x] Version bumped in `version.properties`

### CODE REVIEW

File: WordPress/jetpack_metadata/release_notes.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
-This release is short and sweet: after deleting a site, you might’ve noticed that same site sticking around in your My Site list until you restarted the app.
+* [***] My Site: your My Site screen now has two tabs, "Site Menu" and "Home". Under "Home", you'll find contextual cards with some highlights of whats going on with your site. Check your drafts or scheduled posts, your today's stats or go directly to another section of the app. [https://github.com/wordpress-mobile/WordPress-Android/issues/15989]
+* [*] Site creation: Adds a new screen asking the user the intent of the site [https://github.com/wordpress-mobile/WordPress-Android/pull/16306]
+* [*] Notifications: Fixed notification detail crash for comments with File block [https://github.com/wordpress-mobile/WordPress-Android/pull/16314]
+* [**] Block Editor: Quote block: Adds support for V2 behind a feature flag [https://github.com/WordPress/gutenberg/pull/40133]
```
Comment (AliSoftware) [inline WordPress/jetpack_metadata/release_notes.txt:4]: @geriux I'm not sure I should keep this one entry in the final release notes to end users, given it's behind a feature flag, but I'm also not sure what kind of feature flag that was (one that can be toggled by any users in their prefs? One that the client app chose to toggle on/off when integrating GBM — in which case I'm not sure what WPAndroid used — or something else?

Overall, given (at least as far as I understand this change) this shouldn't have any visible impact for end users, I'm not even sure if we'd include it in the final release notes (that will be editorialized based on this draft list), so I'd be tempted to remove it anyway… though I see you've used a `[**]` priority indicator, so I thought maybe that was still important enough to be kept?

(If we decide to remove it, here's a quick GitHub suggestion to apply the removal quickly)
```suggestion
```

File: WordPress/metadata/release_notes.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
-This release is short and sweet: after deleting a site, you might’ve noticed that same site sticking around in your My Site list until you restarted the app.
+* [***] My Site: your My Site screen now has two tabs, "Site Menu" and "Home". Under "Home", you'll find contextual cards with some highlights of whats going on with your site. Check your drafts or scheduled posts, your today's stats or go directly to another section of the app. [https://github.com/wordpress-mobile/WordPress-Android/issues/15989]
+* [*] Site creation: Adds a new screen asking the user the intent of the site [https://github.com/wordpress-mobile/WordPress-Android/pull/16306]
+* [*] Notifications: Fixed notification detail crash for comments with File block [https://github.com/wordpress-mobile/WordPress-Android/pull/16314]
+* [**] Block Editor: Quote block: Adds support for V2 behind a feature flag [https://github.com/WordPress/gutenberg/pull/40133]
```
Comment (AliSoftware) [inline WordPress/metadata/release_notes.txt:4]: Same question as above

(If we decide to remove it, here's a quick GitHub suggestion to apply the removal quickly)
```suggestion
```

File: WordPress/metadata/release_notes.txt [hunk 3]
```diff
@@ -1,3 +1,7 @@
-This release is short and sweet: after deleting a site, you might’ve noticed that same site sticking around in your My Site list until you restarted the app.
```
Comment (AliSoftware) [inline WordPress/metadata/release_notes.txt:1]: I manually removed the following entry that was automatically extracted from the `RELEASE-NOTES.txt`, given that it was marked `[internal]`:

```
* [*] [internal] Site creation: Adds a new screen asking the user the name of the site [https://github.com/wordpress-mobile/WordPress-Android/pull/16259]
```
@mkevins I believe you marked it as `[internal]` because, from what I read from your associated PR, this was behind a feature flag, so that's not yet visible to end users, right? Just wanted to confirm that me removing this item from what will end up in the 19.7 release notes was appropriate 🙂

**Review by oguzkocer — APPROVED** [review]

**Comment by oguzkocer** [comment]
@AliSoftware Sorry, I didn't realize auto-merge was enabled. I thought I'd approve the PR, so you could merge it as soon as you got answers to your questions, but it ended up merging the PR.

**Comment by AliSoftware** [comment]
Eh, that's my fault, I probably shouldn't enable Auto-Merge on PRs when I have pending questions on them 😓 

@geriux @mkevins I'm still interested in your feedback anyway — depending on your answers we will see if we need to open an additional PR to adjust the release notes accordingly.

File: WordPress/metadata/release_notes.txt (hunk 3 above)
Comment (mkevins) [reply WordPress/metadata/release_notes.txt:1]: Sort of.. this is actually enabled (feature flag default `true`), but is also conditioned with an ExPlat flag which is used remotely to run an experiment (upcoming). The internal note indicates that this should be included in calls for testing, but not in the public release notes (more info on the "internal" label in this Slack thread: p1649059586421809-slack-C033D5RCN2Y ).

tl;dr removing those from the public release notes is correct :+1:  Thank you for double checking!

Comment (AliSoftware) [reply WordPress/metadata/release_notes.txt:1]: Got it, thanks! (And TIL about the meaning we give to `[internal]` in that context!)

cc @thehenrybyrd for info about still including that item in the CfT for WPAndroid 19.7 despite me removing it from the draft for public facing release notes here.

File: WordPress/jetpack_metadata/release_notes.txt (hunk 1 above)
Comment (geriux) [reply WordPress/jetpack_metadata/release_notes.txt:4]: Hey @AliSoftware 👋 this shouldn't be included, I'm sorry it was included in the main apps release notes 😞 

Since this was merged already, what is the process to have it removed? Let me know how I can help with that.

Thanks for asking about this!

Comment (AliSoftware) [reply WordPress/jetpack_metadata/release_notes.txt:4]: Doing a PR to remove them is usually the way to go;

But since @mokagio already pinged our freelance writer in our #wordpress-release-notes in p1650432777950529-slack-C02S5P5MBGA I'd suggest you leave a message to her in that channel ASAP to let her know too that she should omit that item in the public-facing notes.

Comment (AliSoftware) [reply WordPress/jetpack_metadata/release_notes.txt:4]: Btw if that item was originally included in the `RELEASE-NOTES.txt` file so that it gets beta-tested and included in the CfT even if not yet public-facing because behind a feature flag, I recently learned that using `[internal]` for such cases is then usually the way to go (i.e. "`[internal]`" meaning "include in CfT but not public facing notes").

If that's the case for this item here, be sure to let @thehenrybyrd know that she should include the item (or not) in her CfT

Comment (geriux) [reply WordPress/jetpack_metadata/release_notes.txt:4]: Cool! Thanks! I already sent a message there.

Comment (geriux) [reply WordPress/jetpack_metadata/release_notes.txt:4]: TIL about `[internal]` I'll have it in mind for future updates. In this case, we do want to test it because it will be enabled in a few weeks cc @thehenrybyrd

Comment (thehenrybyrd) [reply WordPress/jetpack_metadata/release_notes.txt:4]: @geriux as I understand this can be tested with a self-hosted site with `Quote Block v2` enabled in Gutenberg Experiment settings; I've included this in the internal CfT :)

File: WordPress/metadata/release_notes.txt (hunk 3 above)
Comment (thehenrybyrd) [reply WordPress/metadata/release_notes.txt:1]: @AliSoftware Yep! `[internal]` has been used for that in WPiOS for a while, we're just starting to use in in WPAndroid - I've got a p2 post drafted about it and other release notes best practices, should be up this week :)

File: WordPress/jetpack_metadata/release_notes.txt (hunk 1 above)
Comment (geriux) [reply WordPress/jetpack_metadata/release_notes.txt:4]: > as I understand this can be tested with a self-hosted site with Quote Block v2 enabled in Gutenberg Experiment settings; 

That's correct, awesome! Thank you!

Comment (thehenrybyrd) [reply WordPress/jetpack_metadata/release_notes.txt:4]: Thank you for including that detail in the PR notes!
